pkg/content_manager.go

```diff
@@ -36,14 +37,17 @@ func (cm *ContentManager) pkgMgr() *PackageManager {
 }
 
 func (cm *ContentManager) tmpDir() string {
+	if !cm.instance.manager.aem.config.TemplateFileExists() {
+		return os.TempDir()
+	}
 	return cm.instance.manager.aem.baseOpts.TmpDir
 }
 
-func (cm *ContentManager) Download(localFile string, opts PackageCreateOpts) error {
-	if opts.PID == "" {
-		opts.PID = fmt.Sprintf("aemc:content-download:%s-SNAPSHOT", timex.FileTimestampForNow())
+func (cm *ContentManager) Download(localFile string, packageOpts PackageCreateOpts) error {
+	if packageOpts.PID == "" {
+		packageOpts.PID = fmt.Sprintf("aemc:content-pull:%s-SNAPSHOT", timex.FileTimestampForNow())
```

Comment:
method has a 'download' in its name so package name was accurate; if this should be dynamic then maybe put something (pkg kind or sth) inside `PackageCreateOpts`